src/services/NetworkService.js

```diff
@@ -216,9 +216,19 @@ export default {
       .post(`${API_ROOT}/session/${sessionId}/report`, { reportMessage })
       .then(this._successHandler, this._errorHandler);
   },
-  adminGetSessions(page) {
+  adminGetSessions({
+    page,
+    showBannedUsers,
+    showTestUsers,
+    sessionActivityFrom,
+    sessionActivityTo,
+    minMessagesSent,
+    minSessionLength
+  }) {
     return Vue.http
-      .get(`${API_ROOT}/sessions?page=${page}`)
+      .get(
+        `${API_ROOT}/sessions?page=${page}&showBannedUsers=${showBannedUsers}&showTestUsers=${showTestUsers}&minSessionLength=${minSessionLength}&sessionActivityFrom=${sessionActivityFrom}&sessionActivityTo=${sessionActivityTo}&minMessagesSent=${minMessagesSent}`
```

Comment:
fyi there's a handy built-in API for creating query strings: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/toString

`const queryString = new URLSearchParams({ page, showBannedUsers, showTestUsers }).toString()`

Reply:
Oh wow, this is awesome! Thank you!